composer.json

```diff
@@ -28,9 +28,9 @@
         "laravel/browser-kit-testing": "^6.2",
         "mockery/mockery": "^1.4.2",
         "nunomaduro/collision": "^5.0",
-        "fakerphp/faker": "^1.9.1",
         "facade/ignition": "^2.5",
-        "tightenco/duster": "^0.3.0"
+        "tightenco/duster": "^0.3.0",
+        "fakerphp/faker": "^1.15"
```

Comment:
Faker comes for free with all Laravel apps :) No need to specifically request it.

Reply:
(uhh... sorry.. it normally comes for free in require-dev, but it doesn't here cause this is an OLLLDDD app lol)